src/main/java/io/confluent/connect/hdfs/TopicPartitionWriter.java

```diff
@@ -618,9 +618,15 @@ private void resetOffsets() throws ConnectException {
       // tempfile, but then that tempfile was discarded). To protect against this, even if we
       // just want to start at offset 0 or reset to the earliest offset, we specify that
       // explicitly to forcibly override any committed offsets.
-      long seekOffset = offset > 0 ? offset : 0;
-      log.debug("Resetting offset for {} to {}", tp, seekOffset);
-      context.offset(tp, seekOffset);
+      if (offset > 0) {
+        long seekOffset = offset;
```

Comment:
Now, with this refactoring, seems safe to just use `offset` (it's still a primitive) in this branch. I think it's simpler than declaring `seekOffset`. WDYT?